components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java

```diff
@@ -113,6 +113,24 @@ public static IdentityProviderManager getInstance() {
         return instance;
     }
 
+    /**
+     * Retrieves tenantID.
+     *
+     * @param tenantDomain Tenant name
+     * @return tenantId
+     */
```

Comment:
Incomplete method comment; missing exception details and missing `.` at the end.
If we are adding a comment make sure to complete that.